tests-e2e/cypress/services/institute-api-service.ts

```diff
@@ -88,31 +88,65 @@ export class InstituteApiService {
         return authorityResult?.content[0]?.independentAuthorityId;
     }
 
+    async getAuthorityByAuthorityName(authorityName: string) {
+     
+        const authoritySearchCriteria = [{
+          condition: null,
+          searchCriteriaList: [
+            {
+              key: "displayName",
+              operation: "eq",
+              value: authorityName,
+              valueType: "STRING",
+              condition: "AND"
+            },
+            {
+              key: "closedDate",
+              operation: "eq",
+              value: null,
+              valueType: "STRING",
+              condition: "AND"
+            }
+          ]
+        }];
+  
+        const authoritySearchParam = {
+          params: {
+            searchCriteriaList: JSON.stringify(authoritySearchCriteria)
+          }
+        };
+
+        const url = `${this.config.env.institute.base_url}${AUTHORITY_ENDPOINT}/paginated`;
+        const authorityResult = await this.restUtils.getData(url, authoritySearchParam);
+        return authorityResult?.content[0];
+    }
+
     async createAuthorityWithContactToTest(){
-        let authorityID = await this.getAuthorityIDByAuthorityNumber('998');
+        let authority = await this.getAuthorityByAuthorityName('E2E Automation Testing Authority');
```

Comment:
Could we keep this as EDX instead of E2E. I found it useful to differentiate which authority was being used by Student Admin and which one was EDX. 

E2E could apply to either one.